composer.json

```diff
@@ -5,7 +5,7 @@
     "homepage": "https://airbrake.io",
     "license": "MIT",
     "require": {
-        "php": ">=5.4"
+        "php": "^5.4.7 || ^7.0"
```

Comment:
Frameworks like Symfony and Laravel has chosen `5.5.9` as their minimum requirement. `5.5` will see end of life in just three months.